tests/services/test_encryption_service.py

```
@@ -41,80 +39,80 @@ def test_encrypt_decrypt_file(self, db_session, tmp_path):
+
         file_encryption_service.encrypt_file(db_session, input_file, encrypted_file)
-        
+
         # Corrupt the tag (last 16 bytes)
         data = bytearray(encrypted_file.read_bytes())
-        data[-1] ^= 0xFF 
+        data[-1] ^= 0xFF
         encrypted_file.write_bytes(data)
-        
+
         with pytest.raises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)